.github/workflows/dbt_run.yml

```diff
@@ -16,6 +16,30 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check for base_trades changes
+        id: check_base_trades
+        if: inputs.project == 'dex'
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            git fetch origin main
+            CHANGES=$(git diff --name-only origin/main..HEAD | grep -E "dbt_subprojects/${{ inputs.project }}/models/.*/platforms/.*_base_trades\.sql$" || true)
+          else
+            CHANGES=$(git diff --name-only HEAD~1 | grep -E "dbt_subprojects/${{ inputs.project }}/models/.*/platforms/.*_base_trades\.sql$" || true)
+          fi
+          
+          if [ ! -z "$CHANGES" ]; then
+            echo "has_changes=true" >> $GITHUB_OUTPUT
+            MODIFIED_FILES=$(echo "$CHANGES" | tr '\n' ',' | sed 's/,$//')
+            echo "modified_files=$MODIFIED_FILES" >> $GITHUB_OUTPUT
+            echo "Found changes in base_trades files:"
+            echo "$CHANGES"
+          else
+            echo "has_changes=false" >> $GITHUB_OUTPUT
+            echo "No base_trades changes found"
+          fi
```

Comment:
can we move this step down, just one above the actual test below? keep them next to each other and organized, easier to follow.

Reply:
Moved.